shared/style/value_selector.css

```diff
@@ -4,7 +4,7 @@
 
 /* Main dialog setup */
 [role="dialog"][data-type="value-selector"] {
-  background:rgb(243, 243, 243);/* UX:6.4.1 #2d2d2d --> white*/
+  background:#565050;/* UX:6.4.1 #2d2d2d --> white*/
```

Comment:
Yes, It's not white. It's display the same in VD given. Hex value is good, It's not necessary to fill every color in RGB format. I will remove the absolute comments.